### PR TITLE
Not found page and minor improvements

### PR DESCRIPTION
**Pull request by fairnightzz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,5 +15,9 @@ Start a development server with
 npm start
 ```
 
+For it to read manga, make sure to set up a `config.tsx` file, with
```
Comment (MrPicklePinosaur) [inline README.md:18]: why is the config a .tsx, doesnt matter too much tho

File: src/pages/Reader.tsx [hunk 2]
```diff
@@ -51,6 +51,15 @@ class Reader extends React.Component<any, State> {
     this.myRef = React.createRef();
   }
 
+  mangaName = (): string => {
```
Comment (MrPicklePinosaur) [inline src/pages/Reader.tsx:54]: this is why we might want to store a metadata file along with the actual manga.

what if the name has punctuation and special characters and stuff:
```
darwins_game.zip -> Darwins Game
```

**Review by MrPicklePinosaur — APPROVED** [review]
https://media.tenor.co/videos/4b37971fd6b0a4d88549a1ba7b6a4409/mp4

File: README.md (hunk 1 above)
Comment (fairnightzz) [reply README.md:18]: Trauma from the state thing when trying to implement the state u did in react native - ill move it back to .ts

File: src/pages/Reader.tsx (hunk 2 above)
Comment (fairnightzz) [reply src/pages/Reader.tsx:54]: Do you think I should create metadata now because we're gonna need it later -  maybe ill look into how other ppl do it

Comment (fairnightzz) [reply src/pages/Reader.tsx:54]: o lol just saw the issue

**Pull request merged**